code/modules/awaymissions/zlevel.dm

```diff
@@ -43,6 +43,7 @@ GLOBAL_LIST_INIT(potentialConfigRandomZlevels, generateConfigMapList(directory =
 
 /proc/generateMapList(filename)
```

Comment:
what stops someone doing `../passwd` as the filename instead

Reply:
Byond does, it doesn't work last I checked. If it did then basically every proc that uses file paths is vulnerable to exploitation

Reply:
Just checked, turns out it does work. So now we have 10x the amount of vulnerabilities because variable paths are almost never sanitized

Reply:
yeah well that's why you have to ensure your admins are trustable.